src/main/java/com/google/api/codegen/transformer/ruby/RubyGapicSurfaceTransformer.java

```diff
@@ -399,6 +409,36 @@ private String topLevelPackagePath(SurfaceNamer namer) {
     return Joiner.on(File.separator).join(paths);
   }
 
+  private String postVersionDirPath(SurfaceNamer namer) {
+    List<String> parts = namer.getApiModules();
+    List<String> paths = new ArrayList<>();
+    boolean versionFound = false;
+    for (String part : parts) {
+      if (versionFound) {
+        paths.add(namer.packageFilePathPiece(Name.upperCamel(part)));
+      }
+      if (VersionMatcher.isVersion(part)) {
+        versionFound = true;
+      }
+    }
+    return Joiner.on(File.separator).join(paths);
```

Comment:
Hmm, this is mostly unrelated.. but does it make sense to use `File.separator`? There is no guarantee that the generator is going to be run on the same platform as the actual code and (I don't work on Windows much) but I suspect the paths are still specified with `/`.